docker/test/integration/steps/steps.py

```diff
@@ -248,6 +262,27 @@ def step_impl(context, file_name, content, path):
     context.test.add_test_data(path, content, file_name)
 
 
+@given("a Funnel with the name \"{funnel_name}\" is set up")
+def step_impl(context, funnel_name):
+    funnel = Funnel()
+    funnel.set_name(funnel_name)
```

Comment:
You wrote the ctor that takes the name as an optional argument :)

Reply:
True, updated in f6a40ef8ad7c2e10b048100e8279dfb00a95c14d